src/main/java/marquez/core/services/JobRunService.java

```diff
@@ -0,0 +1,23 @@
+package marquez.core.services;
+
+import java.util.UUID;
+import marquez.api.JobRun;
+import marquez.api.JobRunState;
+import marquez.core.exceptions.UnexpectedException;
+
+public class JobRunService {
```

Comment:
these methods are in `JobService`, so can be removed

Reply:
Removing.